modules/security-deploy-image-sources.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-deploy.adoc
+
+[id="security-deploy-image-sources_{context}"]
+= Controlling what image sources can be deployed
+
+It is important that the intended images are actually being deployed, that the
+images including the contained content
+are from trusted sources, and they have not been altered. Cryptographic signing
+provides this assurance. {product-title} enables cluster administrators to apply
+security policy that is broad or narrow, reflecting deployment environment and
+security requirements. Two parameters define this policy:
+
+- one or more registries (with optional project namespace)
```

Comment:
In general, we try to avoid parentheses if possible. And especially (s). For these bullets, suggest trying to reword :

```
* One or more registries, with an optional project namespace
* A trust type, which can be to accept, reject or require public keys
```

Reply:
Done.